xt/README.txt

```diff
@@ -0,0 +1,5 @@
+Following common practice in CPAN distributions, this directory is for "author tests",
```

Comment:
In our case, as our code is more a program than a module, I think it would make sense to put that test in the /test/ directory, and have it run for all PRs like the unit and integration tests and perlcritic etc.